pages/submarine/guide_nixos.mdx

```diff
@@ -0,0 +1,55 @@
+# Install NixOS with Submarine
+
+Note: While ARM images could technically be built aswell, they are not considered ready and therefore are not supported.
+
+## Prepare an image
+
+1. Either dowmload a `raw-x64cros` image [from CI](https://github.com/ninelore/flake/actions/workflows/images.yml) or follow the [build instructions](https://github.com/ninelore/flake/blob/main/README.md#build-instructions)
+
+2. Write the image to an external media using `dd`.
+
+3. Reminder: to boot Submarine from external media, you first need to run the following commands as root.
+
+```bash
+sudo enable_dev_usb_boot
+```
+
+4. Reboot, and on the Developer Mode screen, press Ctrl+U to boot from external media.
+
+## Partitioning
+
+1. After booting the image, establish network connection with NetworkManager (`nmtui` or `nmcli`).
+
+2. When partitioning the drive, the first partition should be a 16MB partition (64MB on aarch64) for Submarine. You do not need to create a EFI system partition (ESP) as Submarine does not use UEFI.
+
+3. Set the necessary flags for the Submarine partition with `cgpt add -i <partition number> -t kernel -P 15 -T 1 -S 1 /dev/sdX`, substituting &lt;partition number&gt; for the number of the Submarine partition and `/dev/sdX` for the block device of your internal drive.
```

Comment:
does this even render correctly? why not juse use `<>`

Reply:
using `<>` will lead to it being interpreted as a html tag

![image](https://github.com/user-attachments/assets/d4067776-2484-424b-9653-dd4e98db049b)

Reply:
thought that wouldn't work for markdown; the more you know:tm:

Reply:
on second thought: might aswell be fenced code

Reply:
being able to fall back to html is a markdown feature/spec x)